ansible_collections/arista/avd/plugins/plugin_utils/schema/avddataconverter.py

```diff
@@ -112,16 +112,18 @@ def convert_items(self, items: dict, data: list, schema: dict, path: list[str]):
             return
 
         for index, item in enumerate(data):
+            path[-1]+=f"[{index}]"
```

Comment:
This will in-place change change `path`, so you will have an invalid path on the second and following loops. It was added below, to avoid modifying the original list.

Reply:
You probably just need to edit the existing added items to be `+ [index]` instead of a string.

Reply:
it will be like - router_bgp.peer_groups.0.peer_filter

Reply:
Fixed!